.rubocop.yml

```diff
@@ -0,0 +1,3 @@
+inherit_gem:
```

Comment:
Minor: we should add the merge config here too, as standard, to avoid future surprises with custom config for this repo.